test/daimon_skycrawlers/crawler_test.rb

```diff
@@ -3,14 +3,21 @@
 class DaimonSkycrawlersCrawlerTest < Test::Unit::TestCase
   sub_test_case 'fetch html' do
     setup do
-      @crawler = ::DaimonSkycrawlers::Crawler.new 'http://example.com'
+      stubs = Faraday::Adapter::Test::Stubs.new do |stub|
```

Comment:
`Faraday::Adapter::Test::Stubs` does not have such API.

Fadaray can specify base URL such as `http://www.example.com/blog/`.